README.md

```diff
@@ -107,6 +107,16 @@ VAGRANT_PREFER_SYSTEM_BIN=1 vagrant ssh -c "sudo maestro"
 
 ### Local Build
 
+You can also build without using Vagrant.
+
+#### Prerequisites
+
+* Linux
+* go (tested on go 1.15.1)
+* libuv
```

Comment:
should be: libuv1-dev

I also noticed libunwind-dev listed as a dependency in the distro-pelion-edge recipe.  Is that needed here?

Reply:
libunwind was a dependency for libgrease.  It's no longer needed.

Reply:
> should be: libuv1-dev

OK.  Added as a hint for Debian users.